app/src/main/res/values/strings_settings.xml

```diff
@@ -2,19 +2,19 @@
 <resources>
 
     <!-- Settings interface -->
-    <string name="title_settings">Settings / Help</string>
+    <string name="title_settings">Settings and help</string>
     <string name="title_customization">Customization</string>
     <string name="title_appearance_settings">Appearance</string>
     <string name="title_appearance_settings_general">General appearance</string>
-    <string name="title_appearance_settings_applications">Applications appearance</string>
+    <string name="title_appearance_settings_applications">Appearance of apps</string>
     <string name="title_behavior_settings">Operation</string>
-    <string name="title_help_and_general_info">Help and general information</string>
+    <string name="title_help_and_general_info">Help and general info</string>
 
     <!-- Notification -->
-    <string name="set_notification">Notification for favorites access</string>
+    <string name="set_notification">Notification for access to favorites</string>
     <string name="set_notification_help">Uses a transparent icon</string>
-    <string name="notification_title">Access to favorites</string>
-    <string name="notification_popup_title">🔧 Manage favorites</string>
+    <string name="notification_title">Favorites</string>
+    <string name="notification_popup_title"Manage favorites</string>

```

Comment:
```suggestion
    <string name="notification_popup_title">Manage favorites</string>
```
Fixing a typo, but you didn't want to have the 🔧 emoji in this title?

Reply:
```suggestion
    <string name="notification_popup_title">🔧 Manage favorites</string>
```
Option 2 fixing the typo and adding the emoji

Reply:
Good catch. The 🔧 is always the menu only otherwise.